charts/sshpiper/templates/role.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.rbac.create }}
+{{- if and .Values.sshpiper.kubernetes_all_namespaces (not .Values.rbac.clusterRole) }}
```

Comment:
i do not think `.Values.rbac.create` here is necessary

Reply:
Applies same rationale as above.